tm2/pkg/crypto/keys/client/addpkg.go

```diff
@@ -7,12 +7,12 @@ import (
 	"flag"
 	"fmt"
 
+	"github.com/gnolang/gno/gno.land/pkg/sdk/vm"
```

Comment:
addpkg should be defined in gno.land, not tm2.

Reply:
It appears there's many shared struct between the different commands, so moving only `call.go` and `addpkg.go` in `gno.land/cmd/gnokey` is not straightforward.

What about moving the whole `pkg/crypto/keys/client` folder into `gno.land/cmd/gnokey/client` ? I tested and it requires only a few changes in some imports: 

```diff
-	"github.com/gnolang/gno/tm2/pkg/crypto/keys/client"
+	"github.com/gnolang/gno/gno.land/cmd/gnokey/client"
```

Reply:
I believe it's a smart approach: let's start by moving to gno.land and then take the time to design well-crafted reusable components in the right place for future blockchains.

Reply:
As I mentioned above

Reply:
This does not work; please review the discussion and my comment below.

Reply:
Not really generic if I may: one of the first thing it does is fetching the config from `$GNO_HOME` or `$XDG_CONFIG_HOME/gno`.

But I understand your point, it would be better to the `addpkg` and `call` subcommands in the `gnokey` package. I just feel it's a shame to keep this ultimate dependency.

The other way is to simply turn public the related structs (`makeTxCfg`, `queryCfg`) and methods (`signAndBroadcast`, `queryHandler`, ...) . At some point, to make this package more flexible, that makes sense.